### PR TITLE
Add good/bad comparison to Default migration values

### PR DESCRIPTION
**Pull request by syndbg**
Add a good/bad comparison and how-to.

@bbatsov

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -802,9 +802,18 @@ when you need to retrieve a single record by some attributes.
 <sup>[[link](#default-migration-values)]</sup>
 
   ```Ruby
-  # bad - application enforced default value
-  def amount
-    self[:amount] or 0
+  # BAD - application enforced default value
```
Comment (bbatsov) [inline README.md]: BAD -> bad

File: README.md [hunk 2]
```diff
@@ -802,9 +802,18 @@ when you need to retrieve a single record by some attributes.
...
+  class Product < ActiveRecord::Base
+    def amount
+      self[:amount] || 0
+    end
+  end
+
+  # GOOD - database enforced
```
Comment (bbatsov) [inline README.md]: same here

**Comment by syndbg** [comment]
@bbatsov Addressed.

**Comment by olegykz** [comment]
Why not **change_column_default**?
http://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_column_default
(which is reversible in 5.x.x rails)

**Pull request merged**